package.json

```diff
@@ -3,16 +3,21 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
     "@fortawesome/fontawesome-free": "^6.4.0",
+    "@mui/icons-material": "^5.14.0",
```

Comment:
hey we are already using [MDB React](https://mdbootstrap.com/docs/react/) react for bootstraping ,so don’t add mui